CHANGES.md

```diff
@@ -1516,7 +1516,6 @@ Error Messages / Semantic Checks
   (e.g., `var x: int; proc foo(y: x) ...` now generates an error as intended)
 * added a warning for misleading uses of `new`  
   (e.g., `var x: borrowed MyClass = new owned MyClass();`)
-* added an error when fields with runtime types cannot be default-initialized
```

Comment:
I think you wanted to remove line 333, this one might have been something someone else added (at least, I'm pretty sure line 333 was the one you added in the same commit as `SortedSet.kth()`)